Automattic/wp-calypso
# Login: Display inline validation messages in `LoginForm`

<img width="422" alt="screen shot 2017-05-05 at 4 35 26 pm" src="https://cloud.githubusercontent.com/assets/1130674/25767735/e3492c68-31b0-11e7-8768-786da8223fef.png">

This PR:

- Updates `LoginForm` to use properties from the global store instead of properties from `this.state` when possible.
- Updates the selectors/tests for `login.requestError` in the global store to use an object instead of a string.
- Updates `LoginForm` to display inline validation messages.

**Testing**
- Visit `/login`.
- Enter an invalid username/password combination.
- Assert that you see an inline error message like above.
